Build-Scenario.md

```diff
@@ -13,20 +13,20 @@ For specific target-oriented attacks, custom scenarios may be necessary.
   - Description (mandatory): A quick description (<50 words) of the scenario
   - PayloadPath (optional): If the phishing scenario pushes malwares to victims, users can insert the absolute path of  the malicious executable here
 
-  ii) context: This section is optional and holds user-defined variables that may be later injected to the template.
+ii) context: This section is optional and holds user-defined variables that may be later injected to the template.
 
 ####Example
 -------
 
 Here's an example of a config.ini file:
 
- # This is a comment
- [info]
- Name: ISP warning page
- Description: A warning page from victim's ISP asking for DSL credentials
- [context]
- victim_name: John Phisher
- victim_ISP: Interwebz
+ "# This is a comment"<br/>
```

Comment:
Instead of adding a break wrap the whole thing in the "```" like so:
```
# This is a comment
 [info]
 Name: ISP warning page
 Description: A warning page from victim's ISP asking for DSL credentials
 [context]
 victim_name: John Phisher
 victim_ISP: Interwebz
```